core/src/lib.rs

```diff
@@ -80,6 +83,8 @@ pub mod __reexports {
 	cfg_client_or_server! {
 		pub use tokio;
 	}
+
+	pub use super::proc_macros_support::*;
```

Comment:
Is it possible to remove this module the proc macro crate instead?

Reply:
Sorry I'm not sure what you mean here

Reply:
I mean whether it's possible to move the helpers in `core/src/proc_macros_support.rs` to some helper functions inside `proc-macors/srs/utils`  instead?

Reply:
That's not possible, proc-macro crates cannot export anything other than proc macros

Reply:
I meant whether it's possible to return TokenStream2 instead but anyway I'm happy with this approach and looks clean